modules/reactor/include/meshgenerators/PeripheralRingMeshGenerator.h

```diff
@@ -82,4 +96,28 @@ class PeripheralRingMeshGenerator : public PolygonMeshGeneratorBase
    * @return whether the boundary is the external boundary of the given mesh
    */
   bool isExternalBoundary(ReplicatedMesh & mesh, const boundary_id_type bid) const;
+
+  /**
+   * Define node positions of the inner boundary layer that is conformal to the input mesh's
+   * external boundary.
+   * @param input_ext_node_num number of nodes on the external boundary of the input mesh
+   * @param input_bdry_angles list of angles (in rad) formed by three neighboring nodes on the
+   * external boundary of the input mesh
+   * @param ref_inner_bdry_surf Reference outmost layer (surface) points of the inner boundary layer
+   * @param ref_inner_bdry_azi azimuthal angles of the surface points of the reference inner
+   * boundary layer
+   * @param inner_peripheral_bias_terms terms describing the cumulative radial fractions of the
+   * nodes within the inner boundary layer
+   * @param azi_array list of azimuthal angles (in degrees) for radius correction purpose

```

Comment:
azimuthal angles of what? How is this different from ref_inner_bdry_azi ?

Reply:
azi_array is the list of azi angle of of thenodes on input mesh's external boundary.

By moving these nodes outward along the normal direction, we can get `ref_inner_bdry_azi`. So they are different.

Reply:
Will add more description.